packages/fx-core/templates/plugins/resource/aad/auth/tab/README.md

```diff
@@ -46,4 +46,6 @@ To learn more about Teams Toolkit local debug functionalities, refer to this [do
 
 The AAD [manifest](https://docs.microsoft.com/azure/active-directory/develop/reference-app-manifest) allows you to customize various aspects of your application registration. You can update the manifest as needed.
 
-Follow this [document](https://aka.ms/teamsfx-aad-manifest#customize-aad-manifest-template) if you need to include additional API permissions to access your desired APIs.
+Follow this [document](https://aka.ms/teamsfx-aad-manifest#how-to-customize-the-aad-manifest-template) if you need to include additional API permissions to access your desired APIs.
```

Comment:
to access your desired APIs
->
to access your desired permissions

Reply:
Request permission to access a particular API, I think that's correct?